src/pocketmine/lang/Installer/deu.ini

```diff
@@ -0,0 +1,51 @@
+language_has_been_selected = Deutsch wurde erfolgreich als Sprache ausgewählt.
+skip_installer = Möchtest du den Einrichtungsassistenten überspringen?
+
+welcome_to_pocketmine = Willkommen bei Genisys!\nBevor es mit der Einrichtung deines neuen Servers losgehen kann, musst du die Lizenz akzeptieren.\nGenisys ist unter der LGPL Lizenz lizenziert, welche du in der Datei LICENCE in diesem Ordner nachlesen kannst.
```

Comment:
Stimmt, man sollte eben doch nicht zu sehr am Original-Text kleben -> shit in = shit out :smile:
"...that you can read opening the LICENSE file on this folder."
Es würde sicherlich auch reichen "...welche du in der Datei LICENCE nachlesen kannst."